packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -506,6 +506,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
                           className: Dropdown.slotClassNames.clearIndicator,
                           styles: styles.clearIndicator,
                           accessibility: indicatorBehavior,
+                          ...(!search && { tabIndex: 0 }),
```

Comment:
Should we move this logic into `indicatorBehavior` to be more consistent with other components? Right now we work with tabbIndex mostly in Behaviors only (apart from Dropdown and RadioGroup components)

Reply:
I think we should leave indicatorBehavior as it is. Based on this comment and the other one maybe we should create a new behavior, clearIndicatorBehavior or something, that will be tabbable, not have any aria-hidden and instead have aria-label.

@jurokapsiar @kolaps33 what do you think?

Reply:
If I understand it correctly with this change there will be element which will have:
role: img,
aria-hidden: true
tabIndex: 0

What doesn't looks reasonable from my point of view.

I think would be better if clear indicator icon element has:
role: button 
tabIndex: 0

And in this case it would make sense to put in into new behavior.

Reply:
or even wouldn't be there more clear way to use icon only button? 
    <Button icon="icon-close" text iconOnly title="Close" />
just asking as I don't know the code well if it is possible or not... :)